spec/models/registration_spec.rb

```diff
@@ -22,4 +22,28 @@
       end
     end
   end
+
+  describe '#destroy_purchased_tickets' do
+    it 'destroys purchased tickets if tickets are purchased' do
+      user = build(:user)
```

Comment:
then i shouldn't create conference as well... as registration creates it already?